dev-python/importlib_metadata/importlib_metadata-6.8.0.recipe

```diff
@@ -0,0 +1,65 @@
+SUMMARY="A library to access the metadata for a Python package"
+DESCRIPTION="This project primarily supports third-party packages installed \
+by PyPA tools (or other conforming packages). It does not support:
+- Packages in the stdlib.
+- Packages installed without metadata."
+HOMEPAGE="https://pypi.python.org/pypi/importlib-metadata"
+COPYRIGHT="2017-2023 Jason R. Coombs, Barry Warsaw"
+LICENSE="Apache v2"
+REVISION="1"
+SOURCE_URI="https://pypi.python.org/packages/source/i/$portName/$portName-$portVersion.tar.gz"
+CHECKSUM_SHA256=""
+
+ARCHITECTURES="any"
+
+PROVIDES="
+	$portName = $portVersion
+	"
+REQUIRES="
+	haiku
+	"
+
+BUILD_REQUIRES="
+	haiku_devel
+	"
+
+PYTHON_PACKAGES=(python39 python310)
+PYTHON_VERSIONS=(3.9 3.10)
+for i in "${!PYTHON_PACKAGES[@]}"; do
+	pythonPackage=${PYTHON_PACKAGES[i]}
+	pythonVersion=${PYTHON_VERSIONS[$i]}
+
+	eval "PROVIDES_${pythonPackage}=\"
+		${portName}_$pythonPackage = $portVersion
+		\""
+	eval "REQUIRES_$pythonPackage=\"
+		haiku
+		zipp_$pythonPackage
+		cmd:python$pythonVersion
+		\""
+	BUILD_REQUIRES+="
+		build_$pythonPackage
+		installer_$pythonPackage
+		setuptools_$pythonPackage
+		setuptools_scm_$pythonPackage
```

Comment:
Getting a build error with setuptools_scm included on 64bit (doesn't seem to be needed?)